drivers/sensor/iis3dhhc/iis3dhhc_trigger.c

```diff
@@ -146,8 +148,8 @@ int iis3dhhc_init_interrupt(struct device *dev)
 #endif /* CONFIG_IIS3DHHC_TRIGGER_OWN_THREAD */
 
 	ret = gpio_pin_configure(iis3dhhc->gpio, cfg->int_pin,
-			   GPIO_DIR_IN | GPIO_INT | GPIO_INT_EDGE |
-			   GPIO_INT_ACTIVE_HIGH | GPIO_INT_DEBOUNCE);
+				 GPIO_INPUT | GPIO_INT_DEBOUNCE |
```

Comment:
The interrupt signal is generated directly by the IIS3DHHC chip. This is a 'high' quality signal. Should not bounce. I propose to remove `GPIO_INT_DEBOUNCE` flag. The same holds true for other occurrences of  `GPIO_INT_DEBOUNCE` flag in this PR.

Reply:
> The interrupt signal is generated directly by the IIS3DHHC chip. This is a 'high' quality signal. Should not bounce. I propose to remove `GPIO_INT_DEBOUNCE` flag. The same holds true for other occurrences of `GPIO_INT_DEBOUNCE` flag in this PR.

@mnkp 
Yes, I remember a similar comment recently. So, what would you suggest? To remove
debounce from all sensor drivers? I mean, for all 4 sensors I touched in this PR (lps22hh, lsm6dso, iis3dhhc, lis2dw12).

Reply:
Yes, I propose the remove `GPIO_INT_DEBOUNCE` from all drivers.

Reply:
OK, done and re-pushed. Also, I had forgot to change the drdy description in the iis3dhhc bindings, so I added it as well.